client/src/components/organisms/AgendaSection.tsx

```diff
@@ -1,15 +1,30 @@
 import React from "react";
 import { Box, Card } from "@/components/atoms";
 import { SectionHeader } from "@/components/molecules/SectionHeader";
+import { OngoingAgendaCard } from "../molecules/OngoingAgendaCard";
+import { Agenda, OngoingAgenda } from "biseo-interface/agenda";
+import { useAgenda } from "@/services/agenda";
+
+const isOngoingAgenda = (agenda: Agenda): agenda is OngoingAgenda => {
+  return agenda.status === "ongoing";
+};
```

Comment:
이 부분 @kjy2844 와 함께 봤던 type guard 부분인데 나중에 따로 `utils` 폴더로 빼서 공통으로 사용하도록 하는 것이 좋을 것 같아요!
```suggestion
const isOngoingAgenda = (agenda: Agenda): agenda is OngoingAgenda => {
  return agenda.status === "ongoing";
}; // TODO : move to utils
```

Reply:
넵 이건 다른 나머지 agenda 부분 머지되면 한 번 정리해서 utils로 옮기겠습니다.